webpack/scenes/ContentViews/Create/ContentViewCreateActions.js

```diff
@@ -0,0 +1,18 @@
+import { API_OPERATIONS, get } from 'foremanReact/redux/API';
+import api, { orgId } from '../../services/api';
+import CONTENT_VIEWS_KEY from './ContentViewsConstants';
+
+export const createContentViewsParams = extraParams => ({
+  organization_id: orgId(),
+  nondefault: true,
+  ...extraParams,
+});
+
+const createContentView = extraParams => get({
```

Comment:
I know this is still a draft, but here are some small suggestions that hopefully will save some time :) 

is it a `get` or a `post`? Looks like there is a `post` available [here](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/redux/API/APIActions.js#L24). I think you can drop the `type` param too.

There is also an `errorToast` and `successToast` param available [here](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/redux/API/APIRequest.js#L20-L21 ), I think these will be helpful here as well for success and error handling feedback.